torch/_inductor/codegen/common.py

```diff
@@ -501,8 +501,7 @@ def deduce_output_dtype_by_name(
     elif op_name == "reduction":
         return kwargs["dtype"] if "dtype" in kwargs else args[1]
     elif op_name == "constant":
-        dtype = kwargs["dtype"] if "dtype" in kwargs else args[-1]
-        return DTYPE_TO_COMPUTATION_DTYPE[dtype]  # type: ignore[index]
+        return kwargs["dtype"] if "dtype" in kwargs else args[-1]
```

Comment:
Because now `dtype` of constant should be updated inside `legalize_lowp_fp_dtype_loopbody` based on its usage.